nose_parameterized/parameterized.py

```diff
@@ -211,11 +211,53 @@ def expand(cls, input, testcase_func_name=None):
             >>>
             """
 
-        def parameterized_expand_wrapper(f):
+        def doc_expansion(f, args, kwargs, is_method):
+            offset = 1 if is_method else 0
+            vns = inspect.getargspec(f)[0][offset:]
+            descs = ["{0} = {1}".format(n, repr(v)) for n, v in zip(vns, args)]
+            descs.extend(["{0} = {1}".format(k, repr(v)) for k, v in kwargs.items()])
```

Comment:
Please first merge the provided kwargs with the default kwargs then sort those, so the output order will be consistent both between testcases and between test runs (ex, so `def test_foo(self, a=1, b=2)` run with `()`, `a=42`, and `b=42`, will have `a = 1, b = 2`, `a = 42, b = 2`, and `a = 1, b = 42` respectively).